core/src/main/java/org/elasticsearch/index/query/QueryStringQueryBuilder.java

```diff
@@ -266,6 +271,7 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         this.fuzziness.writeTo(out);
         out.writeVInt(this.fuzzyPrefixLength);
         out.writeVInt(this.fuzzyMaxExpansions);
+        out.writeBoolean(this.fuzzyTranspositions);
```

Comment:
Pretty much. You'd want to change the version constant. In your case I'd go with `Version.V_5_4_0_UNRELEASED`. If a constant with that name doesn't exist then create it following the normal pattern in that file. That is the earliest version that will see your change. There isn't really a good way of knowing that version without talking to us though. We decide which versions are safe for backporting based on how releases are going.